src/ripple/protocol/Quality.h

```diff
@@ -234,6 +234,20 @@ class Quality
 
     friend
     bool
+    operator<= (Quality const& lhs, Quality const& rhs) noexcept
```

Comment:
I don't think it is a huge deal either way.  But I don't recommend it.  The benefit is small and the cost is small.  The benefit is that you don't have to write 4 boiler-plate operators.  The cost is that `Quality` now has `boost` as an associated namespace which will be used for ADL.  Also readers have to look into boost docs to make sure that this does exactly what one would guess it does.  And trying to figure that out from reading the boost implementation is virtually impossible.